src/widgets/FormInputs/FormDateInput.js

```diff
@@ -60,8 +60,8 @@ export const FormDateInput = React.forwardRef(
 
     const [inputState, setInputState] = React.useState({
       isValid: true,
-      inputValue: moment(initialValue).format('DD/MM/YYYY'),
-      pickerSeedValue: initialValue.toDate(),
+      inputValue: initialValue.isValid() ? moment(initialValue).format('DD/MM/YYYY') : '',
+      pickerSeedValue: initialValue.isValid() ? initialValue.toDate() : new Date(),
```

Comment:
Another change which is a bit cludgey. See above re. date logic, had to make this change here to allow a blank date input.

Reply:
`initialValue?.isValid()` prob a good idea